### PR TITLE
Add old catalog help page to static site

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to https://github.com/GSA/datagov-deploy/issues/3704

### CODE REVIEW

**Comment by nickumia-reisys** [comment]
Specific page: https://federalist-a1176d2b-cb31-49a0-ba20-a47542ee2ec5.app.cloud.gov/preview/gsa/datagov-website/catalog-help/catalog-help/

**Review by FuhuXia — CHANGES_REQUESTED** [review]
shall we take the chance to update some of the links?

**Comment by FuhuXia** [comment]
should we take the chance to update the links?
[Suggest a new dataset](https://explore.data.gov/nominate) and see what others have suggested
Contribute code or [suggest a change](https://github.com/GSA/data.gov/issues?labels=&milestone=&page=1&state=open) on Github
Ask a general question at the [Open Data Stack Exchange](https://opendata.stackexchange.com/) and tag it with “data.gov”
Send an [email](https://www.data.gov/contact#question)

File: www.data.gov/catalog-help.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+NOTE: when going to a specific dataset, you may be redirected to an organization’s web site, rather than getting a direct download.
+
+
+### Collaborating
+
+<p>There are many ways to provide feedback and contribute to the discussions on Data.gov and this catalog.</p>
+- [Suggest a new dataset](https://explore.data.gov/nominate) and see what others have suggested
+- Contribute code or [suggest a change](https://github.com/GSA/data.gov/issues?labels=&milestone=&page=1&state=open) on Github
+- Ask a general question at the [Open Data Stack Exchange](https://opendata.stackexchange.com/) and tag it with “data.gov”
+- Send an [email](https://www.data.gov/contact#question)
```
[the remark this answers is not in the document]
Comment (jbrown-xentity) [reply www.data.gov/catalog-help.md:33]: Agreed. We don't monitor stack exchange, so remove that one. Also remove suggest a new dataset.
Add the data.gov/contact form. Keep email. change contribute code or suggest change link to the https://github.com/gsa/datagov-deploy repo, unless @mogul still wants to change the names...

[the remark this answers is not in the document]
Comment (nickumia-reisys) [reply www.data.gov/catalog-help.md:33]: Sure.. did you have specific updates in mind?  I'm not sure which links aren't relevant anymore.

**Review by jbrown-xentity — APPROVED** [review]

**Review by FuhuXia — APPROVED** [review]

**Pull request merged**